docs/releases/2.1.0.rst

```diff
@@ -1,20 +1,21 @@
 ================================================
-RapidSMS 2.1.0 release notes (under development)
+RapidSMS 2.1.0 release notes (current release)
```

Comment:
The "(current release)" note will need to be removed from 2.0.0 as well. Refer to the previous release notes docs to see how they look.